src/attributes.md

```diff
@@ -0,0 +1,49 @@
+# Attributes
+
+Attributes come in two types - 'builtin'/'inert' attributes, and 'non-builtin'/'active' attributes:
```

Comment:
I feel like the single quotes here and in the headings make the text a bit harder to read; are you okay with removing them?

-----

> They are 'inert', meaning they are left as-is by the macro expansion code.

This one seems fine to keep, but I feel that the others make it harder to read.